lib/kubeclient.rb

```diff
@@ -39,6 +39,73 @@ def handling_kube_exception
       raise KubeException.new(e.http_code, JSON.parse(e.response)['message'])
     end
 
+    def get_all_for_entity(entity)
```

Comment:
Yeah...do you have a better idea?  Maybe just `get_entities` to match the previous `get_pods`, `get_nodes`, etc.?